### PR TITLE
Add Makefile with sync designation

### LINKED ISSUE #50: Sharing `lib` assets
As we've moved to a mono-repo for the different variants, we need to replicate the `lib` folder to each subfolder (`inst/`, `_extensions/countdown/assets`, `src/assets`). 

Probably a short bash script to copy if changes are detected in `lib` or just a script for when a release is scheduled to make sure the `lib` assets are up-to date in the relevant area.

### PR DESCRIPTION
**Pull request by coatless**
Close #50 by implementing a `make sync`/`make`.

### CODE REVIEW

**Comment by coatless** [comment]
Thinking about this more, I'll probably go back to the more "verbose" version in https://github.com/gadenbuie/countdown/pull/55/commits/eb9cec77d93a2570b5ac98ff857e75cb862b364e instead of the symbolic rules to simplify the version injection.

**Comment by gadenbuie** [comment]
I think that's a good idea, it was also my first impression looking at the PR, but I hadn't had time to write back yet. I think it'll be a bit more approachable and we may need slightly different processes for version updating anyway.

**Comment by coatless** [comment]
Okidokie, we have the version information being set in: `lib/version.txt`. 

This information is then written into an appropriate `config.{lua,R,py}` file. From there, the version string is used in the relevant HTML dependency attachment.

Did you want the version information to be written into the files via a comment? Or do you think the `config` file approach would be sufficient enough? 

(P.S. The `quarto` CSS has local changes for RevealJS support that probably should go into the `lib` directory.)

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+SYNC_DEST_R := r/inst/countdown
+SYNC_DEST_QUARTO := quarto/_extensions/countdown/assets
+
+sync-all: sync-python sync-r sync-quarto ## Sync web assets to all subpackages
+
+sync-python: ## Sync web assets to Python package
+	@echo "Syncing web assets to the Python package..."
+	@test -d $(SYNC_DEST_PYTHON) || mkdir -p $(SYNC_DEST_PYTHON)
+	cp -r $(SRC_FILES) $(SYNC_DEST_PYTHON)
+	echo "countdown_version = '$(VERSION)'" > python/countdown/config.py
```
[the remark this answers is not in the document]
Comment (coatless) [reply Makefile:28]: Okidokie, we could always do: `asset_version` or the more verbose `countdown_asset_version` if need be later on.

**Review by coatless — COMMENTED** [review]
The `make help` is a neat trick! 

Thanks for adding in `debug` and simplifying the `SRC_FILES` merge. 

Feel free to merge in whenever you have a moment.

**Pull request merged**